src/TorchSharp/Tensor/Tensor.cs

```diff
@@ -42,30 +60,35 @@ internal Tensor(IntPtr handle)
             public override bool Equals(object? obj)
             {
                 return (obj is Tensor) && this.Equals((obj as Tensor)!);
-
             }
 
             /// <summary>
             ///  TBD
             /// </summary>
             /// <returns></returns>
-            public override int GetHashCode()
-            {
-                return base.GetHashCode();
-            }
+            public override int GetHashCode() => base.GetHashCode();
 
             /// <summary>
             /// A friendly name for the tensor. This is useful for debugging purposes.
             /// </summary>
             public string? name { get; set; }
 
+            /// <summary>
+            /// Helper method to determine if a tensor is disposed or not.
+            /// </summary>
+            public bool IsDisposed => handle == IntPtr.Zero;
```

Comment:
Linguistically speaking, this should be 'WasDisposed' but I'm not sure I can think of any precedent for using past tense in public properties.

I'm not asking for a change, I'm just curious about other APIs using anything but present tense.

Reply:
Heh, I sometimes struggle with using Are instead of Is

Reply:
Use present tense. It is not much different from `IsEnabled` in that it tells the current state, not necessarily that some specific action took place. If you want to be really picky, rename it to `IsInvalid`.